plugins/dynamix.vm.manager/VMMachines.page

```diff
@@ -101,7 +101,7 @@ if ($subaction) {
   }
   echo "<script>clearHistory();</script>";
 }
-if ($libvirt_running=='yes') $vms = $lv->get_domains();
+if ($libvirt_running=='yes') $vms = $lv->get_domains() ?: [];
```

Comment:
I think the same fix will need to be applied to this line too?: https://github.com/limetech/webgui/blob/a5329592b598fafcb8f93102ba21d8a871413494/plugins/dynamix.vm.manager/include/VMMachines.php#L25

Reply:
yeah I'll make that change, thx!